src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCosts = 40;
+  const totalCost = dayCosts * days;
+  const baseDiscount = 20;
```

Comment:
Consider using more descriptive variable names than `baseDiscount` and `largeDiscount`. For example, `discountForThreeDays` and `discountForSevenDays`. 🤔